.travis.yml

```diff
@@ -41,4 +41,14 @@ matrix:
           fi
         - docker build -t asegroup1/backend-$DOCKER_BRANCH:latest .
         - docker push asegroup1/backend-$DOCKER_BRANCH:latest
-
+    - language: java
+      jdk:
+        - oraclejdk8
```

Comment:
Technically we will be running the project on openjdk but I assume oracle would work